sqlalchemy_ibmi/base.py

```diff
@@ -42,6 +42,9 @@
   Defaults to ``False``.
 * ``readonly`` - If ``True``, the connection is set to read-only. Defaults to ``False``.
 * ``timeout`` - The login timeout for the connection, in seconds.
+* ``use_system_naming`` - If ``False``, the connection is set to use the SQL
+   naming convention, otherwise it will use the System naming convention.
+   Defaults to ``True``.
```

Comment:
The default here is wrong as we default to False (using SQL naming). Additionally, it reads rather oddly, sort of describing the opposite of what it does.  It should read something like (abbrev): 'If true, uses System naming mode, otherwise uses SQL naming mode...'